<?xml version="1.0"?> 
<sdf version="1.4">
  <world name="target_spawn_world">

    <!-- Ground Plane -->
    <include>
      <uri>model://ground_plane</uri>
    </include>

    <include>
      <uri>model://sun</uri>
    </include>

    <!-- Target 0 -->
    <model name="target_0">
      <pose>0 0 0 0 0 0</pose>
      <link name="link">
        <collision name="collision">
          <geometry>
            <cylinder>
              <radius> 0.00001 </radius>
              <length> 0.4 </length>
            </cylinder>
          </geometry>
        </collision>

        <visual name="visual">
          <geometry>
            <cylinder>
              <radius> 0.2 </radius>
              <length> 0.4 </length>
            </cylinder>
          </geometry>
          <material>
            <ambient>0 0 1 1</ambient>
            <diffuse>0 0 1 1</diffuse>
            <specular>0.1 0.1 0.1 1</specular>
            <emissive>0 0 0 0</emissive>
          </material>
        </visual>
      </link>

      <plugin name="animated_box_journal" filename="libanimated_box_journal.so"/>
    </model>

    <!--<model name="target_0_next_pose">
      <pose>0 0 0 0 0 0</pose>
      <link name="link">
        <collision name="collision">
          <geometry>
            <box>
              <size> 0.01 0.01 0.01 </size>
            </box>
          </geometry>
        </collision>

        <visual name="visual">
          <geometry>
            <box>
              <size> 0.2 0.2 0.2 </size>
            </box>
          </geometry>
          <material>
            <ambient>0 0 1 1</ambient>
            <diffuse>0 0 1 1</diffuse>
            <specular>0.1 0.1 0.1 1</specular>
            <emissive>0 0 0 0</emissive>
          </material>
        </visual>
      </link>

      <plugin name="animated_box_journal" filename="libanimated_box_journal.so"/>
    </model>-->

    
  </world>
</sdf>
